.prettierrc

```diff
@@ -10,6 +10,6 @@
     "semi": true,
     "singleQuote": true,
     "tabWidth": 4,
-    "trailingComma": "es5",
+    "trailingComma": "none",
```

Comment:
I understand your point of view, because I was strictly against trailing commas some time ago as well. But Prettier changed the default value of this rule from "none" to "es5", which includes trailing commas in multiline arrays and objects only ("all" would insert them kind of everywhere, which would be inappropriate in my opinion).
I personally noticed "es5" as sensible, because apparently I did change the order of elements in multiline arrays/objects quite often, so it was pretty tedious to manually append a comma to the previously last element and remove it from the new last element.
MDN says:
> Trailing commas (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.

Reply:
Easier moving and cleaner adding of lines are very good points 👍